From 89eb448b6c5941921acdc7e8b5e18050900e9f20 Mon Sep 17 00:00:00 2001
Message-Id: <89eb448b6c5941921acdc7e8b5e18050900e9f20.1624424665.git.pmatilai@redhat.com>
In-Reply-To: <9772abc058ae7183dbfb2356ec0af0a3d610c2c9.1624424665.git.pmatilai@redhat.com>
References: <9772abc058ae7183dbfb2356ec0af0a3d610c2c9.1624424665.git.pmatilai@redhat.com>
From: Florian Festi <ffesti@redhat.com>
Date: Tue, 22 Jun 2021 16:53:13 +0200
Subject: [PATCH 2/2] Don't depend on translation sub directories

just create them. Otherwise their date messes up the man page
genenration and triggers a rebuild when on wanted (e.g. when building
form the tar ball)

Resolves: #1729
---
 docs/man/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/man/Makefile.am b/docs/man/Makefile.am
index 595444ad5..e10959255 100644
--- a/docs/man/Makefile.am
+++ b/docs/man/Makefile.am
@@ -87,7 +87,7 @@ man_sk_man8_DATA = sk/rpm.8
 EXTRA_DIST += sk/rpm.8.md
 EXTRA_DIST += sk/rpm.8
 
-%: $(srcdir)/%.md fr ja ko pl ru sk
+%: $(srcdir)/%.md
 	@$(MKDIR_P) `dirname $@`
 	${PANDOC} -s -t man $(srcdir)/$@.md -o $@
 
-- 
2.31.1

